package.json

```diff
@@ -30,6 +30,7 @@
   "private": true,
   "scripts": {
     "start": "react-native start",
+    "start:after-pull": "yarn install && (cd ios && pod install) && yarn generate && yarn start",
```

Comment:
As specified in the README file, the correct command to install the pods is `bundle exec pod install`